### PR TITLE
Tokenize the ASM bytecode view

### LINKED ISSUE #40: [feature request] Tokenization / remapping in ASM view
Currently the Bytecode mode does not contain any tokenization or remapping capabilities, preventing any mapping from being done while using that "decompiler" mode. In some cases, being able to map based on disassembly would be beneficial.

### PR DESCRIPTION
**Pull request by IotaBread**
Fixes #40. Only thing left is to display the constructors of mapped classes as `<init>`s

image

### CODE REVIEW

File: enigma/src/main/java/org/quiltmc/enigma/impl/source/bytecode/EnigmaTextifier.java [hunk 1]
```diff
@@ -1,18 +1,522 @@
...
+			public boolean shouldSkip() {
+				return !this.descriptor.contains("L");
+			}
+		}
+
+		record MethodDescriptor(String descriptor) implements QueuedToken {
+		}
+
+		record Signature() implements QueuedToken {
+		} // TODO
```
Comment (ix0rai) [inline enigma/src/main/java/org/quiltmc/enigma/impl/source/bytecode/EnigmaTextifier.java:48]: stray todo?

**Review by ix0rai — APPROVED** [review]
looks amazing!

**Comment by ix0rai** [comment]
note that this should be merged into either master or a develop/2.1, not dev/2.0

File: enigma/src/main/java/org/quiltmc/enigma/impl/source/bytecode/EnigmaTextifier.java (hunk 1 above)
Comment (IotaBread) [reply enigma/src/main/java/org/quiltmc/enigma/impl/source/bytecode/EnigmaTextifier.java:48]: Oh yeah, let me remove it

**Pull request merged**